README.md

```diff
@@ -83,13 +113,18 @@ To align with [Inboard](https://inboard.bws.bio/), Poetry has been deprecated in
 
 ## Help needed
 
-This stack is in an experimental state, so there is no guarantee for bugs or issues. Please open an issue ticket against this repository to make us aware of issues and we will do our best to respond to them in a timely manner. Please leave feedback on features that would be very beneficial for developers who often leverage mongodb in their FastAPI stack.
+This stack is in an experimental state, so there is no guarantee for to not find bugs or issues. Please open an issue ticket against this repository to make us aware of issues and we will do our best to respond to them in a timely manner. Please leave feedback on features that would be very beneficial for developers who often leverage mongodb in their FastAPI stack.
 
 
 ## Release Notes
 
+For each release of the generated apps, please assume any changed introduced as the potential to be a backwards-breaking change.
```

Comment:
I don't quite understand what this is saying, can you clarify?

Reply:
I changed the language here. Let me know your thoughts.